src/components/nimbleAutoComplete/StyledWrappers.tsx

```diff
@@ -67,7 +61,8 @@ const SlectedIcon = styled(CheckCircleOutlineIcon)(({chipcolor}: SlectedIconProp
 }));
 
 const TextInput = styled(TextField)(({fontFamily}: TextInputProps) => ({
+  '& input': {minWidth: '75px !important'},
```

Comment:
THis is to make sure that "input search" move to new line if it doesn't have enough space(otherwise it'll just show part of the text hidden and it doesn't look good)